src/regions/RankerRegion.js

```diff
@@ -0,0 +1,47 @@
+import { types } from 'mobx-state-tree';
+
+import NormalizationMixin from '../mixins/Normalization';
+import RegionsMixin from '../mixins/Regions';
+import { RankerBoardModel } from '../tags/object';
+import { guidGenerator } from '../core/Helpers';
+import Registry from '../core/Registry';
+import { AreaMixin } from '../mixins/AreaMixin';
+import { AnnotationMixin } from '../mixins/AnnotationMixin';
+
+const Model = types
+  .model('RankerBoardRegionModel', {
+    id: types.optional(types.identifier, guidGenerator),
+    pid: types.optional(types.string, guidGenerator),
+    type: 'rankerboardregion',
+    object: types.late(() => types.reference(RankerBoardModel)),
+  })
+  .volatile(() => ({
+    hideable: true,
+  }))
+  .views(self => ({
+    get parent() {
+      return self.object;
+    },
+
+  }))
+  .actions(() => ({
+
+    serialize() {
+      //TODO remove
+      console.log(self);
```

Comment:
that should throw an error

Reply:
Ah, it's resolved as current window, but anyway should be removed.
Also you can leave some comment about serialization, like "all data is generated in result, but this method will be invoked anyway, so return empty object"